logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -151,4 +159,32 @@ class ConversationDataSource(
                 wrapApiRequest { clientApi.listClientsOfUsers(it) }.map { memberMapper.fromMapOfClientsResponseToRecipients(it) }
             }
     }
+
+    override suspend fun createOne2OneConversationWithTeamMate(otherUserId: UserId): Either<CoreFailure, ConversationId> = wrapApiRequest {
+        conversationApi.createNewConversation(
+            createConversationRequest = CreateConversationRequest(
+                qualifiedUsers = listOf(idMapper.toApiModel(otherUserId)),
+                access = emptyList(),
+                accessRole = listOf(),
+                convTeamInfo = null,
+                messageTimer = null,
+                receiptMode = 0,
+                conversationRole = "",
+                protocol = null,
+                name = ""
+            )
+        )
+    }.map { idMapper.fromApiModel(it.id) }
```

Comment:
Repository should insert the new conversation into the DB after creating it.

Reply:
This is overlapping with my PR we can reuse things

Reply:
You should we able to delegate to the  `createGroupConversation()` method I added in https://github.com/wireapp/kalium/pull/366/files#diff-7cc6befc172b3bde25bc0dae25c48a8c3b54476ec722bd1be49d9b82359693df

Reply:
Since you are creating a team one-to-one you should assign the `convTeamInfo` field though.

Reply:
@typfel awesome, waiting for your merge than :)

Reply:
> @typfel awesome, waiting for your merge than :)

Just waiting for approvals ;)